osu.Game/Beatmaps/Drawables/Cards/BeatmapCardContentBackground.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Overlays;
+
+namespace osu.Game.Beatmaps.Drawables.Cards
+{
+    public class BeatmapCardContentBackground : ModelBackedDrawable<IBeatmapSetOnlineInfo>
```

Comment:
Is there a reason for making this a full `ModelBackedDrawable` as opposed to a `DelayedLoadUnloadWrapper`? We may want this in the future for pooling support maybe, but more curious if there is an immediate reason to have this here.

Could potentially simplify things if it isn't requried, and set the `IBeatmapSetOnlineInfo` in the `ctor`.